README.md

```diff
@@ -33,6 +33,14 @@ docker run -ti -v $HOME/Work/github/zephyr:/workdir \
 docker.io/zephyrprojectrtos/zephyr-build:latest
 ```
 
+The image is also available on hub.docker.com, so you can pull the image
```

Comment:
isnt the above `docker run` pulling the image from docke hub already? Why do you need to login for that?